### PR TITLE
Fix "*" for role descriptions.

### PR DESCRIPTION
**Pull request by KingPanda360**
Fixed words being replaced by stars because if Innersloth's inappropriate words finder. Schizophrenic, Masochist, and Bloodlust can't be fixed unless they are renamed to something else.

### CODE REVIEW

File: Resources/Lang/en_US.json [hunk 1]
```diff
@@ -743,7 +743,7 @@
...
   "LudopathInfoLong": "(Impostors):\nAs the Ludopath, your kill cooldown is randomized.\n\nMinimum it can be is 1 second, while the maximum is your default kill cooldown.",
   "GodfatherInfoLong": "(Impostors):\nAs the Godfather, you vote someone to make them your target.\nIn the next round, if someone kills the target, the killer will turn into a Refugee.",
-  "ChronomancerInfoLong": "(Impostors):\nAs the Chronomancer, you have a charge bar which indicates when the massacre is ready. When it is at 100% the next time you kill someone, you go into massacre mode, meaning you can kill indefinitely until your bar runs out of charge. Otherwise, you have a normal KCD.",
+  "ChronomancerInfoLong": "(Impostors):\nAs the Chronomancer, you have a charge bar which indicates when the slaughter is ready. When it is at 100% the next time you kill someone, you go into slaughter mode, meaning you can kill indefinitely until your bar runs out of charge. Otherwise, you have a normal KCD.",
```
Comment (Ultradragon005) [inline Resources/Lang/en_US.json:746]: also change it in here 
image

File: Resources/Lang/en_US.json [hunk 2]
```diff
@@ -762,7 +762,7 @@
...
   "SnitchInfoLong": "(Crewmates):\nAfter the Snitch completes all tasks, they can see the Impostor's names displayed in red on the meeting. When the Snitch has only one task left, the Impostors will see a 「★」 mark next to the name of themselves and the Snitch. When a Snitch becomes a Madmate, the 「★」 mark turns red.",
   "MarshallInfoLong": "(Crewmates):\nAs the Marshall, complete your tasks to reveal yourself to the rest of the crew.\nOther teams will not be able to see you.\nHowever, madmates CAN see you.",
-  "SpeedBoosterInfoLong": "(Crewmates):\nSpeed Booster increases their movement speed every time they complete a task. Note: due to technical limitations, the Speed Booster appears at a normal speed to others, so they look like a glitch.",
+  "SpeedBoosterInfoLong": "(Crewmates):\nSpeed Booster increases their movement speed every time they finish a task. Note: due to technical limitations, the Speed Booster appears at a normal speed to others, so they look like a glitch.",
```
Comment (Ultradragon005) [inline Resources/Lang/en_US.json:765]: LOL that is an old ahhh role not currently in use

File: Resources/Lang/en_US.json (hunk 1 above)
Comment (KingPanda360) [reply Resources/Lang/en_US.json:746]: Fixed

**Comment by Tommy-XL** [comment]
Damn

**Comment by Tommy-XL** [comment]
Next time it is advisable to use one Pull requests because it leads to conflicts

**Comment by KingPanda360** [comment]
But I thought I only did one PR for this. My previous PR got merged 2 days ago so I couldn't commit this into my previous one. So I had to create another pr for this.

File: Resources/Lang/en_US.json [hunk 3]
```diff
@@ -2592,9 +2592,9 @@
   "SoulCollectorSelfVote": "Host settings do not allow you to collect your own soul",
 
   "ChronomancerKillCooldown": "Ability Charge Time",
-  "ChronomancerDecreaseTime": "Massacre Decrease Time (lower is faster)",
-  "ChronomancerStartMassacre": "MASSACRE: ACTIVATED",
-  "ChronomancerVisionMassacre": "Vision When In Massacre",
+  "ChronomancerDecreaseTime": "Slaughter Decrease Time (lower is faster)",
+  "ChronomancerStartMassacre": "SALUGHTER: ACTIVATED",
```
Comment (Ultradragon005) [inline Resources/Lang/en_US.json:2596]: typo

Comment (KingPanda360) [reply Resources/Lang/en_US.json:2596]: It's the "oops" commit

**Pull request merged**